addon/components/frost-button.js

```diff
@@ -100,7 +122,15 @@ export default Ember.Component.extend({
    * ['small', 'medium', 'large', 'extra-large']
    * @type {String}
    */
-  size: 'medium',
+  size: '',
```

Comment:
It looks like this constitutes a breaking API change and therefore warrants a `#MAJOR#` version-bump